[agent/logs/docker] Document the docker tailing from file feature (#9789)
DataDog/documentation: content/en/agent/amazon_ecs/logs.md

{{% /tab %}}
{{< /tabs >}}

> **Note:** `DD_LOGS_CONFIG_DOCKER_CONTAINER_USE_FILE` requires the Datadog Agent v6.27.0/7.27.0+, the option will be silently ignored for older version.

Reviewer: ```suggestion
**Note:** `DD_LOGS_CONFIG_DOCKER_CONTAINER_USE_FILE` requires the Datadog Agent v6.27.0/7.27.0. This option is silently ignored in older versions.
```
